api/prisma/migrations/20230313145217_add_confirm_affiliations/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Publication" ADD COLUMN     "confirmNoAffliations" BOOLEAN DEFAULT FALSE;
```

Comment:
*Affiliations

Reply:
You can install this VSCode extension that will help you with typos https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker